src/utils/blueprints/get-version.ts

```diff
@@ -3,19 +3,19 @@ import { decideVersion } from '@codemod-utils/blueprints';
 import type { Options } from '../../types/index.js';
 
 const latestVersions = new Map([
-  ['@babel/core', '7.22.6'],
-  ['@babel/plugin-proposal-decorators', '7.22.6'],
+  ['@babel/core', '7.22.10'],
+  ['@babel/plugin-proposal-decorators', '7.22.10'],
   ['@babel/plugin-transform-class-properties', '7.22.5'],
   ['@babel/preset-typescript', '7.22.5'],
-  ['@babel/runtime', '7.22.6'],
-  ['@embroider/addon-dev', '3.1.1'],
-  ['@embroider/addon-shim', '1.8.5'],
+  ['@babel/runtime', '7.22.10'],
+  ['@embroider/addon-dev', '4.1.0'],
```

Comment:
It's possible that installing `@embroider/addon-dev@4.1.0` may introduce breaking changes to the codemod.

- https://github.com/embroider-build/embroider/releases/tag/v4.0.0-addon-dev
- https://github.com/embroider-build/embroider/releases/tag/v4.1.0-addon-dev